src/theme/_header.scss

```
@@ -20,6 +20,9 @@
       flex-grow: 2;
       align-self: center;
       justify-content: center;
+      @media only screen and (max-width: $largest-mobile-screen) {
```

@danalvrz The hamburger menu is also shown at tablet widths so I think this should be
```suggestion
      @media only screen and (max-width: $computer-width) {
```